### PR TITLE
Document argument true/false preferences

### PR DESCRIPTION
**Pull request by jjshoe**
##### ISSUE TYPE

- Docs Pull Request
##### SUMMARY

doc true/false over yes/no

### CODE REVIEW

File: docsite/rst/developing_modules.rst [hunk 1]
```diff
@@ -384,6 +384,8 @@ Include it in your module file like this::
     # ... snip ...
     '''
 
+If an argument takes both C(True)/C(False) and C(Yes)/C(No) the documentation should use C(True) and C(False).
```
Comment (docschick) [inline docsite/rst/developing_modules.rst]: add a comma after C(Yes)/C(No)

File: docsite/rst/developing_modules.rst [hunk 2]
```diff
@@ -498,6 +500,7 @@ Module checklist
     * Does module use check_mode? Could it be modified to use it? Document it
     * Examples: make sure they are reproducible
     * Return: document the return structure of the module
+    * If an argument takes both C(True)/C(False) and C(Yes)/C(No) the documentation should use C(True) and C(False).
```
Comment (docschick) [inline docsite/rst/developing_modules.rst]: add a comma after C(Yes)/C(No)

**Comment by docschick** [comment]
minor editing (comma) suggested noted. otherwise, looks good to me.

**Comment by jjshoe** [comment]
@docschick - Updated.

**Pull request merged**